Aquality.Selenium.Template/Aquality.Selenium.Template/Utilities/BrowserUtils.cs

```diff
@@ -0,0 +1,16 @@
+﻿using OpenQA.Selenium;
+using static Aquality.Selenium.Browsers.AqualityServices;
+
+namespace Aquality.Selenium.Template.Utilities
+{
+    public static class BrowserUtils
+    {
+        public static void OpenInNewTabAndSwitch(string url)
```

Comment:
Maybe we can remove this method and just pass the parameter `switchToNew: true`?  It's already exist in Aquality.Selenium:
![image](https://user-images.githubusercontent.com/9014926/206708465-26d29def-dab3-4d36-a246-5dedaad1e27a.png)